rg_instructor_analytics/views.py

```diff
@@ -396,26 +389,63 @@ def post(self, request, course_id):
         return JsonResponse(data=result)
 
 
-class InstructorAnalyticsFragmentView(EdxFragmentView):
+class GradebookView(AccessMixin, View):
+    """Api for question statistic."""
+
+    def get_grades_values(self, grade_info):
+        """Return percent value of the student grade."""
+        result = [int(g['percent'] * 100.0) for g in grade_info['section_breakdown']]
+        result += [int(grade_info['percent'] * 100.0)]
```

Comment:
I would use append method for adding one element to the list

Reply:
change